lerna.json

```diff
@@ -1,7 +1,6 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "version": "3.0.0-alpha.0",
-  "useNx": true,
```

Comment:
why is this removed?

Reply:
oh yea it does, but I upgraded to the new version of lerna and it told me to get rid of this